internal/core/context.go

```diff
@@ -95,3 +95,7 @@ func ReloadClient(ctx context.Context) error {
 	meta.Client, err = createClient(meta.BuildInfo, "")
 	return err
 }
+
+func ExtractProfileName(ctx context.Context) string {
```

Comment:
I think a better version of this exist on master

Reply:
Did'nt found it :thinking: